src/components/FeatureStore/featureStore.util.js

```diff
@@ -433,8 +436,15 @@ export const navigateToDetailsPane = (
   const { name, tag, iter } = match.params
   let content = []
 
-  if (match.params.pageTab === FEATURE_SETS_TAB && featureSets.length > 0) {
-    content = parseFeatureSets(featureSets)
+  if (
+    match.params.pageTab === FEATURE_SETS_TAB &&
+    featureSets.allData.length > 0
+  ) {
+    if (featureSets.selectedRowData.content[name]) {
+      content = featureSets.selectedRowData.content[name]
+    } else {
+      content = featureSets.allData
+    }
```

Comment:
This is equivalent to:
```suggestion
    content = featureSets.selectedRowData.content[name] || featureSets.allData
```